README.md

```diff
@@ -1,250 +1,138 @@
-    departments {
-        id
-        name
-        employees {
-            edge {
-                node {
-                    id
-                    name
-                    department {
-                        # Just an example of nested relationships
-                        id
-                        name
-                    }
-                }
-            }
-        }
-    }
-}
-"""
-```
+    def departments(self) -> List[DepartmentType]:
+        return session.scalars(select(Department)).all()
 
-## Limitations
-
-SQLAlchemy Models -> Strawberry Types and Interfaces are expected to have a consistent
-(customizable) naming convention. These can be configured by passing `model_to_type_name`
-and `model_to_interface_name` when constructing the mapper.
-
-Natively supports the following SQLAlchemy types:
-
-```python
-Integer: int,
-Float: float,
-BigInteger: BigInt,
-Numeric: Decimal,
-DateTime: datetime,
-Date: date,
-Time: time,
-String: str,
-Text: str,
-Boolean: bool,
-Unicode: str,
-UnicodeText: str,
-SmallInteger: int,
-SQLAlchemyUUID: uuid.UUID,
-VARCHAR: str,
-ARRAY[T]: List[T] # PostgreSQL array
-JSON: JSON # SQLAlchemy JSON
-Enum: (the Python enum it is mapped to, which should be @strawberry.enum-decorated)
+schema = strawberry.Schema(query=Query)
+mapper.finalize()  # Finalize all mapped types
 ```
 
-Additional types can be supported by passing `extra_sqlalchemy_type_to_strawberry_type_map`,
-although support for `TypeDecorator` types is untested.
-
-Association proxies are expected to be of the form `association_proxy('relationship1', 'relationship2')`,
-i.e., both properties are expected to be relationships.
-
-Roots of polymorphic hierarchies **are supported**, but are also expected to be registered via
-`strawberry_sqlalchemy_mapper.interface()`, and its concrete type and
-its descendants are expected to inherit from the interface:
-
-```python
-class Book(Model):
-    id = Column(UUID, primary_key=True)
-
-
-class Novel(Book):
-    pass
-
-
-class ShortStory(Book):
-    pass
-
-
-# in another file
-strawberry_sqlalchemy_mapper = StrawberrySQLAlchemyMapper()
-
-
-@strawberry_sqlalchemy_mapper.interface(models.Book)
-class BookInterface:
-    pass
-
-
-@strawberry_sqlalchemy_mapper.type(models.Book)
-class Book:
-    pass
-
-
-@strawberry_sqlalchemy_mapper.type(models.Novel)
-class Novel:
-    pass
-
-
-@strawberry_sqlalchemy_mapper.type(models.ShortStory)
-class ShortStory:
-    pass
-```
+## Feature Status
+
+### Core Functionality
+- [x] Auto-mapping for columns/relationships
+- [x] Automatic N+1 query prevention
+- [x] SQLAlchemy 1.4 & 2.0 support
+- [x] Type customization via `__exclude__` and `__rename__`
+
+### Performance
+- [x] <1ms overhead per type generation
+- [x] Caching for generated types
+- [ ] Lazy type initialization (in progress)
+
+### Integrations
+- [x] GraphiQL Debug Toolbar
+- [x] Strawberry Federation support
+- [ ] Apollo Tracing (planned)
+
+## Complete Reference
+
+### Type Support
```

Comment:
Please update this table with all the values and keep the information about additional types, I think we should maintain all the code examples and descriptions, what you think?

![image](https://github.com/user-attachments/assets/9a0f9223-d86e-46c6-958f-8dcbc7e34df8)